src/deriver/reasoner/explicit.py

```
@@ -0,0 +1,349 @@
+                "\n".join(new_peer_card),
+                "blob",
+            )
+            async with tracked_db("deriver.update_peer_card") as db:
+                await crud.set_peer_card(
+                    db,
+                    self.ctx[0].workspace_name,
+                    new_peer_card,
+                    observer=self.observer,
+                    observed=self.observed,
+                )
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)